spec/models/manageiq/providers/vmware/infra_manager/vm_spec.rb

```diff
@@ -132,25 +132,6 @@
     end
   end
 
-  context "snapshotting_memory_allowed?" do
-    context "when powered on" do
-      let(:power_state) {{:raw_power_state => power_state_on}}
-
-      it "is allowed" do
-        vm.update(power_state)
-        expect(vm.snapshotting_memory_allowed?).to be_truthy
-      end
-    end
-
-    context "when powered off" do
-      let(:power_state) {{:raw_power_state => power_state_suspended}}
-      it "is not allowed" do
-        vm.update(power_state)
-        expect(vm.snapshotting_memory_allowed?).to be_falsy
-      end
-    end
-  end
-
```

Comment:
@skateman do we need to delete these specs? Could we just change them to check `vm.supports_snapshot_memory?` instead of `vm.snapshotting_memory_allowed?`?